### PR TITLE
ci: deleting docker images from cicd

### PR DESCRIPTION
**Pull request by clatapie**
Docker images need to be cleaned not to overload the VM.
This PR add this step within the CICD.

### CODE REVIEW

**Review by a-bouth — DISMISSED** [review]

File: .github/workflows/ci_cd.yml [hunk 1]
```diff
@@ -192,6 +192,12 @@ jobs:
               docker rm $dockerContainers
           }
 
+      - name: "Delete the Docker images (and untagged ones)"
+        if: always()
+        run: |
+          docker image rm ghcr.io/ansys/ansys-sound:latest
```
Comment (RobPasMue) [inline .github/workflows/ci_cd.yml:198]: Are you sure you always want to delete the image? I wouldn't do that. Just delete untagged images.
```suggestion
```

File: .github/workflows/ci_cd.yml [hunk 2]
```diff
@@ -256,6 +262,12 @@ jobs:
...
```
Comment (RobPasMue) [inline .github/workflows/ci_cd.yml:268]:
```suggestion
```

**Comment by RobPasMue** [comment]
Deleting the image itself is a bit unnecessary and time consuming for future runs (since you will ALWAYS have to download it again). Just remove untagged images (which are the old ones) and leave the latest tag so that it can be reused. Next time a latest tag is pushed to the Docker registry, you will pull the new one and the old one will be untagged.

**Review by RobPasMue — APPROVED** [review]
LGTM!

**Review by anssmolla — APPROVED** [review]

**Review by ansaminard — APPROVED** [review]

**Pull request merged**